libs/build-nss-desktop.sh

```diff
@@ -57,8 +57,8 @@ fi
 # We do not know how to cross compile these, so we pull pre-built versions from NSS CI
 # https://github.com/mozilla/application-services/issues/962
 if [[ "${CROSS_COMPILE_TARGET}" =~ "darwin" ]]; then
-  #https://treeherder.mozilla.org/jobs?repo=nss&revision=7f352e7b7ab682c1245a2dc02b723cc2bf527c47
-  curl -sfSL --retry 5 --retry-delay 10 -O "https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/_enROqNsT52PqGPSbGzRzQ/runs/0/artifacts/public/dist.tar.bz2"
+  #From https://firefox-ci-tc.services.mozilla.com/tasks/index/app-services.cache.level-1.content.v1.nss-artifact/latest
+  curl -sfSL --retry 5 --retry-delay 10 -O "https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/app-services.cache.level-1.content.v1.nss-artifact.latest/artifacts/public%2Fdist.tar.bz2"
```

Comment:
Maybe the only issue here is that the URL should end with `/artifacts/public/dist.tar.bz2`?

Reply:
Do we want to be pulling from a level-1 source here? I think we'd want to go with the level-3 fetch?

Reply:
ah yes! That was exactly what I was looking for to ensure we had something that latest longer (1000 years lol). Will update accordingly